test/suites/ubuntu-core-edgex-image/test.sh

```diff
@@ -10,3 +10,14 @@ do
   ssh "$USER@localhost" -p 8022 "curl -s http://localhost:$port/api/v2/ping; printf '\n'"
 done
 
+# verify that the security is avaliable as a snap option of edgexfoundry within the emulator
+ssh "$USER@localhost" -p 8022 "snap get edgexfoundry security -d; printf '\n'"
+
+# check the status of the device-virtual service within the emulator
+ssh "$USER@localhost" -p 8022 "snap services edgex-device-virtual; printf '\n'"
+
+# verify that device-virtual has the startup message set from the gadget within the emulator
+ssh "$USER@localhost" -p 8022 'snap logs -n=all edgex-device-virtual | grep "Startup message"; printf '\n''
+
+# access the service endpoints via API Gateway outside of the emulator
+curl --insecure --show-err https://localhost:8443/core-data/api/v2/ping
```

Comment:
```
$ curl --insecure --show-err https://localhost:8443/core-data/api/v2/ping
{"message":"Unauthorized"}
```

Reply:
I'm surprised that this request even works. With security disabled, the reverse proxy should not be running.